app/refinements/to_boolean.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Provides a way to cast to boolean
+#
+# Returns true if given any string or symbol that, stripped and downcased, matches:
+#   1. True values recognized by YAML.load: %w[y, yes, true, on] (https://yaml.org/type/bool.html)
+#   2. The opposites of ActiveRecord::Type::Boolean::FALSE_VALUES -- %w[false, 0, f, off]
+# Returns nil if given nil or a blank string
```

Comment:
Curious if anyone has thoughts about returning `nil` given a `nil` or blank?

Reply:
Since we're using this in the context of parsing form parameters, I also suspect this is the behavior we want.